src/components/mobile-menu-container/components/mobile-user-disclosure/index.tsx

```diff
@@ -80,7 +81,10 @@ const MobileUserDisclosure = ({
 				<Text asElement="span" className={s.label} size={100} weight="semibold">
 					{label}
 				</Text>
-				<ul className={s.list}>{items.map(renderItem)}</ul>
+				<ul className={s.list}>
+					{items.map(renderItem)}
+					<UserDropdownDisclosureThemeSwitcher />
```

Comment:
❓ since one is display none depending on the size of the screen is this still an issue?

Reply:
📌💭 To answer your question for future reference: since `display: none` does not remove an element from the HTML, this does still matter. All `id`s in a page's HTML must be unique, regardless of whether they're visible or not.